tests/unit/utils/txn_parser/test_get_order_book_changes.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+import json
+from unittest import TestCase
+
+from xrpl.utils import get_order_book_changes
+
+path_to_json = "tests/unit/utils/txn_parser/transaction_jsons/"
+with open(path_to_json + "offer_created.json", "r") as infile:
+    offer_created = json.load(infile)
+with open(path_to_json + "offer_partially-filled_and_filled.json", "r") as infile:
+    offer_partially_filled_and_filled = json.load(infile)
+with open(path_to_json + "offer_cancelled.json", "r") as infile:
+    offer_cancelled = json.load(infile)
+with open(path_to_json + "offer_with_expiration.json", "r") as infile:
+    offer_with_expiration = json.load(infile)
+
+
+class TestGetOrderBookChanges(TestCase):
+    def test_offer_created(self: TestGetOrderBookChanges):
+        actual = get_order_book_changes(offer_created["meta"])
+        expected = [
+            {
+                "account": "rJHbqhp9Sea4f43RoUanrDE1gW9MymTLp9",
+                "offer_changes": [
+                    {
+                        "flags": 131072,
+                        "taker_gets": {"currency": "XRP", "value": "44.930000"},
+                        "taker_pays": {
+                            "currency": "USD",
+                            "issuer": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
+                            "value": "14.524821",
+                        },
```

Comment:
So to clarify, in this case, this means that the account `rJH...` is receiving 44.93 XRP in exchange for 14.5... USD?

Reply:
(Is the "taker" in this case the account in question?)

Reply:
The account `rJH...` which created the offer is offering 44.93 XRP in exchange for 14.5... USD. The taker is always the one consuming an offer. In other words `taker_gets` amount is always the amount the creator is offering and the `taker_pays` amount is always the amount the creator is receiving.

Reply:
Ah, nevermind, I think taker is the person who accepts this offer, and this account is then the maker of this offer since this has the "created" status